src/actions/chatActions.js

```diff
@@ -0,0 +1,16 @@
+export const CHAT_ADD_MESSAGE = 'CHAT_ADD_MESSAGE';
+export const CHAT_SET_USER_STATUS = 'CHAT_SET_USER_STATUS';
+export const CHAT_CLEAR_MESSAGES = 'CHAT_CLEAR_MESSAGES';
+export const CHAT_SET_USERNAME = 'CHAT_SET_USERNAME';
```

Comment:
@Spetastian This prefixing scheme makes sense to me. Is this common for Redux action names?